app/posts/govuk-eleventy-plugin.md

```diff
@@ -0,0 +1,51 @@
+
+This includes support for [Markdown](https://www.markdownguide.org), a lightweight markup language for formatting plain text documents. Posts can incorporate metadata like publication dates and authors, and be organised into different collections.
+
+With this foundation in place, [the design history](https://bat-design-history.netlify.app/) grew to over 500 posts written by more than 20 authors. In 2020 we made this tool available as [a template repository on GitHub](https://github.com/x-govuk/govuk-design-history) so that other teams could quickly begin writing their own design histories. The tool has since been adopted by other teams, both inside the Department for Education and across government.
+
+However, as the number of websites using this template has grown, the harder it has become to share improvements. Abstracting the common parts into an npm module makes it easier for us to do this.
+
+## A plugin for Eleventy
+
+The [GOV.UK Eleventy Plugin](https://x-govuk.github.io/govuk-eleventy-plugin/) makes it easy to start writing documentation rather than spend time building a website for it.
+
+It does this by providing [a set of layouts](https://x-govuk.github.io/govuk-eleventy-plugin/layouts/) for different content types and uses [typography classes from the GOV.UK Design System](https://design-system.service.gov.uk/styles/typography/) to ensure documents look familiar to users of other GOV.UK services. (This is achieved via [a plugin for markdown-it](https://github.com/x-govuk/markdown-it-govuk), which can be used on other projects if desired).
+
+Markdown written using [an extended syntax](https://x-govuk.github.io/govuk-eleventy-plugin/markdown-advanced/) is also supported, meaning authors can easily include tables, abbreviations, footnotes and code examples in their posts, too.
+
+Knowing that teams often need to write documentation for internal users, or build websites that don't meet [the criteria for using the GOV.UK logo and GDS Transport typeface](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk), the plugin enables the following can be customised:
+
+* Organisation name and logo
+* Font family
+* Brand colour
+* Licence and copyright information
+* Site icons and Open Graph share images
+
+![Examples of customised websites using the plugin.](/assets/posts/govuk-eleventy-plugin/customisation-examples.png)
+
+## Get started
+
+The plugin is already being used by different teams across government who have built marketing pages, blogs, glossaries – and design histories, of course.
+
+One of the benefits of static websites is they don't need a database, which means they can be hosted pretty much anywhere. Popular platforms for hosting static websites include [GitHub Pages](https://pages.github.com) and [Netlify](https://www.netlify.com).
```

Comment:
@frankieroberto Possibly a bit too late in the post, but reworded this section to add a bit of colour about what a static website is.